Sources/SwiftTemplate.swift

```diff
@@ -3,9 +3,16 @@
 // Copyright (c) 2017 SwiftGen
 // MIT Licence
 //
-
+import Foundation
 import Stencil
 
+#if os(Linux)
+#if swift(>=3.1)
```

Comment:
Can you add it to the rakefile? `swift build && swift test`
My `rake test` is broken for some reason (I really really really hate ruby version shizzle)